src/lib/viewers/controls/annotations/AnnotationsControls.tsx

```diff
@@ -54,12 +58,23 @@ export default function AnnotationsControls({
     }, [annotationMode, onAnnotationModeEscape]);
 
     // Prevent empty group from being displayed
-    if (!showHighlight && !showRegion) {
+    if (!showDrawing && !showHighlight && !showRegion) {
         return null;
     }
 
     return (
         <div className="bp-AnnotationsControls">
+            <AnnotationsButton
+                data-resin-target="draw"
+                data-testid="bp-AnnotationsControls-drawBtn"
+                isActive={annotationMode === AnnotationMode.DRAWING}
+                isEnabled={showDrawing}
+                mode={AnnotationMode.DRAWING}
+                onClick={handleModeClick}
+                title={__('drawing_comment')}
```

Comment:
Where is this i18n string located?

Reply:
I don't think we have any translations yet for this string, could you describe the process for how we would generate the translations for it?

Reply:
We need to add the English string to the `src/i18n/en-US.properties` file for the translation process to be kicked off.

Reply:
Thanks for describing the process!